menu-api/Controllers/IngredientController.cs

```diff
@@ -11,49 +12,68 @@ public class IngredientController : ControllerBase
     {
         private readonly IIngredientRepository ingredientRepository;
```

Comment:
Use a different way of naming private readonly fields. In this case, it should be _ingredientRepository